scripts/commitizen.js

```diff
@@ -1,30 +1,30 @@
 // prettier-ignore
 const scopes = [
-  { value: 'angular',       name: 'angular:         anything Angular specific' },
-  { value: 'core',          name: 'core:            anything Nx core specific' },
-  { value: 'bundling',      name: 'bundling:        anything bundling specific (e.g. rollup, webpack, etc.)' },
-  { value: 'detox',         name: 'detox:           anything Detox specific' },
-  { value: 'devkit',        name: 'devkit:          devkit-related changes' },
-  { value: 'express',       name: 'express:         anything Express specific' },
-  { value: 'graph',         name: 'graph:           anything graph app specific' },
-  { value: 'js',            name: 'js:              anything TS->JS specific' },
-  { value: 'linter',        name: 'linter:          anything Linter specific' },
-  { value: 'misc',          name: 'misc:            misc stuff' },
-  { value: 'nest',          name: 'nest:            anything Nest specific' },
-  { value: 'nextjs',        name: 'nextjs:          anything Next specific' },
-  { value: 'node',          name: 'node:            anything Node specific' },
-  { value: 'nx-cloud',      name: 'nx-cloud:        anything NxCloud specific' },
-  { value: 'nx-plugin',     name: 'nx-plugin:       anything Nx Plugin specific' },
-  { value: 'nx-dev',         name: 'nx-dev:         anything related to docs infrastructure' },
-  { value: 'react',         name: 'react:           anything React specific' },
-  { value: 'react-native',  name: 'react-native:    anything React Native specific' },
-  { value: 'expo',          name: 'expo:            anything Expo specific' },
-  { value: 'repo',          name: 'repo:            anything related to managing the repo itself' },
-  { value: 'storybook',     name: 'storybook:       anything Storybook specific' },
-  { value: 'testing',       name: 'testing:         anything testing specific (e.g. jest or cypress)' },
-  { value: 'vite',          name: 'vite:            anything Vite specific' },
-  { value: 'web',           name: 'web:             anything Web specific' },
-  { value: 'webpack',       name: 'webpack:         anything Webpack specific' },
+  { value: 'angular', name: 'angular:         anything Angular specific' },
```

Comment:
Prettier does this unfortunately

Reply:
Jason updated this in a separate PR, can you rebase and undo changes to this file?

Reply:
Done, thanks. I need to check why my prettier makes those changes.